Add instruction to remove workflow (#368)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/cloud/operator/developing-workflows.adoc

The `REASON` field gives you a cue about the current workflow status.
====

You can make changes to the workflow YAML using any Kubernetes editor. For example, you can use `kubectl` and the following command:

[source,bash,subs="attributes+"]
----
kubectl edit workflow/greeting -n <your_namespace>
----
and changing the workflow definition inside the CustomResource Spec section.

> Otherwise you can save the CustomResource definition file and edit it with your desired editor and re-applying it.

Reviewer: ```suggestion
Otherwise, you can save the CustomResource definition file and edit it with your desired editor, and re-applying it.
```

==== Feature Not Yet Supported 
The {operator_name} is under active development. Sometimes a feature might not be available yet. Please see xref:cloud/operator/known-issues.adoc[] for a comprehensive list of available features.

> If you identify you're referring to a feature not yet available, please file a new issue so we can prioritize it for you or ask in {product_name} communication channels.

Reviewer: ```suggestion
If you identify you are referring to a feature that is not available yet, you can file a new issue so we can prioritize it for you or request in {product_name} communication channels.
```
